Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM mesosphere/mesos-modules-dev:latest
+MAINTAINER Mesosphere <support@mesosphere.io>
+
+ADD . /serenity
+
+WORKDIR /serenity
+
+# Install the picojson headers
+RUN wget https://raw.githubusercontent.com/kazuho/picojson/v1.3.0/picojson.h -O /usr/local/include/picojson.h
```

Comment:
Better way to do this?  Maybe it should be included in the upstream `mesosphere/mesos-modules-dev` image?  Or maybe it's OK for now :)